types/src/common_params.rs

```diff
@@ -523,6 +569,21 @@ mod tests {
         assert_eq!(deserialized.min_num_of_validators, 0);
         assert_eq!(deserialized.delegation_threshold, 0);
         assert_eq!(deserialized.min_deposit, 0);
+
+        assert_eq!(
+            Into::<Params>::into(deserialized),
```

Comment:
If you swap the first and the second parameter, you can rewrite it `deserialized.into()` because the compiler can infer the converted type.